app/forms/gpo_verify_form.rb

```diff
@@ -22,7 +22,7 @@ def submit
     if result
       pending_profile&.remove_gpo_deactivation_reason
 
-      if profile_has_pending_in_person_enrollment?
+      if user.has_establishing_in_person_enrollment_safe?
```

Comment:
I'm sorry, I know we've gone back and forth over this a dozen times now, but why is it important to prefer the `user` as the source of truth about the enrollment rather than the `pending_profile`?

We take care to [associate the enrollment with the profile](https://github.com/18F/identity-idp/blob/cfd46292156c858df5439f65eab18e0f65ab6dbc/app/services/idv/session.rb#L73) when the profile is created. Even if we were 100% certain that `user.establishing_in_person_enrollment` was the same as `pending_profile.in_person_enrollment`, I would still prefer to ask the profile. That's why we've explicitly made that association, so there's no chance of confusion.

Reply:
Logically, nothing here has changed. Currently, `GpoVerifyForm#pending_profile` defers to the user anyway in `User#pending_profile` (see https://github.com/18F/identity-idp/blob/main/app/forms/gpo_verify_form.rb#L55), this change simply attempts to remove the middleman. The new method `User#has_establishing_in_person_enrollment_safe?` still prefers the `User#pending_profile` as the source of truth over `User#establishing_in_person_enrollment` at https://github.com/18F/identity-idp/pull/9052/commits/0bacc8e45d2a880eaa445624f50c1814ecfccd19#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR212 as you're advocating here.